src/main/java/io/orkes/conductor/client/http/api/TaskResourceApi.java

```diff
@@ -2088,4 +2100,20 @@ private ApiResponse<String> updateTask1WithHttpInfo(
         Type localVarReturnType = new TypeToken<String>() {}.getType();
         return apiClient.execute(call, localVarReturnType);
     }
+
+    private String getIdentity() {
+        String serverId;
+        try {
+            serverId = InetAddress.getLocalHost().getHostName();
+        } catch (UnknownHostException e) {
+            serverId = System.getenv("HOSTNAME");
+        }
+        if (serverId == null) {
+            serverId =
+                    (EC2MetadataUtils.getInstanceId() == null)
+                            ? System.getProperty("user.name")
+                            : EC2MetadataUtils.getInstanceId();
+        }
+        return serverId;
+    }
```

Comment:
At the moment, it looks like the only way the user has to disable this functionality is by providing a workerID themselves. Unless customers are already doing this, they'll start forwarding more data to Conductor when they upgrade.

Automatically exposing worker hostname and system properties like this could have security implications in some environments. At the very least, customers may want to be aware that we've started doing this in their code. We may want to be more conservative and make automated workerIDs something that the user has to enable explicitly.

Reply:
I get the feedback, but it was basically copy pasted from OSS (https://github.com/Netflix/conductor/blob/main/client/src/main/java/com/netflix/conductor/client/worker/Worker.java#L68-L83). Should we make it differently?

Reply:
> I get the feedback, but it was basically copy pasted from [OSS client](https://github.com/Netflix/conductor/blob/main/client/src/main/java/com/netflix/conductor/client/worker/Worker.java#L68-L83).
Thanks for bringing this up. I wasn't aware that we have a precedent for this.

> Should we make it differently?
Probably not. As long as the release notes are clear that this is being turned on by default, I don't think we'll run into any trouble.